src/sonic-config-engine/config_samples.py

```diff
@@ -75,6 +75,42 @@ def generate_t1_sample_config(data):
         port_count += 1
     return data
 
+def generate_t1_smartswitch_sample_config(data):
+    data = generate_t1_sample_config(data)
+    data['DEVICE_METADATA']['localhost']['subtype'] = 'SmartSwitch'
+
+    mpbr_prefix = '169.254.200'
+    mpbr_address = '{}.254'.format(mpbr_prefix)
+
+    bridge_name = 'bridge_midplane'
+    data['MID_PLANE_BRIDGE'] = {
+        'GLOBAL': {
+            'bridge': bridge_name,
+            'address': '{}/24'.format(mpbr_address)
```

Comment:
@oleksandrivantsiv This field `address` is inconsistent with yang model https://github.com/sonic-net/sonic-buildimage/blob/master/src/sonic-yang-models/yang-models/sonic-smart-switch.yang#L42. Should it be `ip_prefix`?

Reply:
@yaqiangz you are right. I'll create a PR with the fix.